app/assets/stylesheets/custom.scss

```diff
@@ -155,6 +157,40 @@ th {
   text-align: center;
 }
 
+// footer
+footer {
+  background-color: #bcdefa;
+  border-top: 1px solid #eaeaea;
+  bottom: 0;
+  height: 60px;
+  position: relative;
+  width: 100%;
+}
+
+footer a {
+  color: #674388;
+  margin-right: 15px;
+  font-size: 1.3em;
+}
+
+footer a:hover {
+  color: #222;
+}
+
+footer small {
+  float: left;
+  font-size: 0.9em;
+}
+
+footer ul {
+  float: right;
+}
+
+footer ul li {
```

Comment:
Merge rule `footer ul li` with rule on line 161<br>Selector should have depth of applicability no greater than 2, but was 3